README.md

```diff
@@ -1,32 +1,264 @@
+  <code>sudo service postgresql status</code>
+  <p>This might produce the following output:</p>
+  <img src="https://curriculum-content.s3.amazonaws.com/pe-mod-3/setup-postgres/postgres-status.png"/>
+  <p>Now go back and switch to the <code>postgres</code> user and try the <code>psql</code> command again.</p>
+
+</details>
+
+The `psql` command will take us into the PostgreSQL space:
+
+```bash
+postgres=#
+```
+
+Once in this space, double-check the version by typing:
+
+```sql
+SELECT version();
+```
+
+Ensure this prints out the correct version we installed! While we are in this
+space, let's change password. For demonstration purposes, we have the password
+here set as "postgres". If you use a different password, make sure to remember
+it!
+
+```sql
+ALTER USER postgres PASSWORD 'postgres';
+```
+
+To quit, enter `\q`. This will lead us back to our `postgres` user space. To get
+out of the`postgres` user space, type `exit`.
+
+Next we want to open up PostgreSQL so our host machine can access it through
+pgAdmin.
+
+Navigate to the `/etc/postgresql/15/main` directory and, as the `sudo`
+user, open up the `postgresql.conf` file:
+
+```bash
+sudo vi postgresql.conf
+```
+
+Once in the file, search for "list_addresses". It may be commented out.
+Un-comment it out and change the line to the following:
+
+```text
+listen_addresses = '*'
+```
+
+Within this file, note the port that PostgreSQL will be running on. By default,
+this should be 5432. If it is a different port number, take note of
+it, as we will need this in a later section. Before saving the file, check the
+changes made with the screenshot below:
+
+![postgresql.conf-changes](https://curriculum-content.s3.amazonaws.com/pe-mod-3/setup-postgres/postgresql-conf-file-changes.png)
+
+We'll need to make one more change so that our host machine can access the
+database on our virtual machines. Open up the `pg_hba.conf` file:
+
+```bash
+sudo vi pg_hba.conf
+```
+
+Search for "IPv4 local connections" inside the file. We may see a line under
+that comment that looks like this:
+
+```text
+host    all             all             127.0.0.1/32            scram-sha-256
+```
+
+Copy and paste that line below it and change the IP address to your computer's
+IP address along with the method. For example:
+
+```text
+host    all             all             10.0.0.149/32            trust
+```
+
+If you don't know your IP address, you can find it by opening a terminal window
+on your host machine and type `ifconfig`. Note: You may need to use the IPv4
+address listed under "Wireless LAN adapter Wi-Fi" as the VM is connected to the
+network through a bridged adapter. Before saving the file, check the changes
+align with the text below:
+
+```text
+host    all             all             127.0.0.1/32            scram-sha-256
+host    all             all             10.0.0.149/32            trust
```

Comment:
@kcleland0818 I agree with including `0.0.0.0/0` simply for the sake of avoiding issues. Using their real IP instead could be problematic because the student could switch networks (which would lease them a different IP), or the router itself could re-lease a different IP for the same client.

I would just include it with the following disclaimer as a note:
> **Note:** Adding `host all all 0.0.0.0/0 md5` to the PostgreSQL configuration file will allow *any* client to connect to the database server using any user account, as long as the client can provide the correct password. While this can be convenient for development or testing purposes, it is generally not recommended for production environments due to several potential security issues. In a real production environment, we should either whitelist an IP or use a different method of connection like [SSH Tunnels](https://www.postgresql.org/docs/8.3/ssh-tunnels.html).

Reply:
Good note! Okay, I will add that then.